src/main/java/com/xceptance/xlt/mastercontroller/Main.java

```diff
@@ -422,21 +443,75 @@ private static CommandLine parseCommandLine(final String[] args)
     private static void printUsageInfoAndExit(final Options options)
     {
         final HelpFormatter formatter = new HelpFormatter();
-        formatter.setSyntaxPrefix("Usage: ");
-        formatter.setWidth(79);
 
+        System.out.println("Usage:");
+        System.out.println("    mastercontroller [<other options>]");
+        System.out.println("      -> Runs in interactive mode. Choose the command to be executed next.");
         System.out.println();
-        formatter.printHelp(XltConstants.MASTERCONTROLLER_EXECUTABLE_NAME + " [<options>]", "\nOptions:", options, "");
+        System.out.println("    mastercontroller -c <commandList> [<other options>]");
+        System.out.println("      -> Runs in non-interactive mode. Pass the commands to be executed.");
+        System.out.println();
+        System.out.println("    mastercontroller -auto [<other options>]");
+        System.out.println("      -> Runs a load test in non-interactive mode by executing all needed commands automatically.");
+        System.out.println();
```

Comment:
Why not using the formatter for printing that as was done before? It's a little bit weird to format and print the program usage text manually but use the formatter to format and print the program's options.

Reply:
True. Fixed.